infrastructure/aws/ecs.tf

```diff
@@ -0,0 +1,53 @@
+data "aws_caller_identity" "current" {
+}
+
+module "my-ecs" {
+  source         = "github.com/in4it/terraform-modules//modules/ecs-cluster"
+  vpc_id         = module.vpc.vpc_id
+  cluster_name   = "loono-dev-api-cluster"
+  instance_type  = "t2.small"
+  ssh_key_name   = aws_key_pair.mykeypair.key_name
+  vpc_subnets    = join(",", module.vpc.public_subnets)
+  enable_ssh     = true
+  ssh_sg         = aws_security_group.allow-ssh.id
+  log_group      = "my-log-group"
+  aws_account_id = data.aws_caller_identity.current.account_id
+  aws_region     = var.AWS_REGION
+}
+
+module "my-service" {
+  source              = "github.com/in4it/terraform-modules//modules/ecs-service"
```

Comment:
Wrong path double-slash
"github.com/in4it/terraform-modules`//`modules/ecs-service"

Reply:
Also, please review the rest